src/shared/lib/CopyNumberUtils.ts

```diff
@@ -0,0 +1,22 @@
+export const ALTERATIONS = {
```

Comment:
/src/pages/patientView/copyNumberAlterations/column/CnaColumnFormatter.tsx

Checkout out my solution in this file.  Is there a reason why it's deletion here and deep deletion there?  

Also, see use of ENUM.  Could possibly serve in place of switch statement.  Maybe we cansolidate code?